src/args.ts

```diff
@@ -69,4 +69,34 @@ export let args : ArgDescriptor[] = [
       ' Defaults to "index.html".',
     type: String,
   },
+  {
+    name: 'protocol',
+    alias: 'h',
```

Comment:
We're using `h` as an alias for `help` in the CLI. Things are easier if there aren't any clashes. How about `P`?